verticapy/plotting/_plotly/__init__.py

```diff
@@ -15,5 +15,12 @@
 permissions and limitations under the License.
 """
 from verticapy.plotting._plotly.bar import BarChart
-from verticapy.plotting._plotly.pie import PieChart
-from verticapy.plotting._plotly.barh import HorizontalBarChart
\ No newline at end of file
+from verticapy.plotting._plotly.pie import PieChart, NestedPieChart
+from verticapy.plotting._plotly.barh import HorizontalBarChart
+from verticapy.plotting._plotly.base import PlotlyBase
+
+import plotly.io as pio
+import plotly.graph_objects as go
+
+pio.templates["VerticaPy"] = go.layout.Template(layout_colorway=PlotlyBase.get_colors())
```

Comment:
what are pio templates?

Reply:
these basically set the color/theme of the plotly library. This way, i do not need to input colors to all the different plotly functions independently.

Reply:
Nice.